fiftyone/utils/labels.py

```
@@ -1198,3 +1205,223 @@ def _get_filepath(sample_or_frame, sample):
+            camera_model,
+            width,
+            height,
+            sample,
+        )
+        if ply:
+            polylines.append(ply)
+
+    return polylines
+
+
+def _process_detection_to_ply(
```

Pylint (3.3.7)

[refactor] Too many positional arguments (8/5)

(R0917)